spec/example_app/app/assets/stylesheets/base/_variables.scss

```diff
@@ -0,0 +1,104 @@
+// Variables for the GrowPath UI application
```

Comment:
I was wondering where all this CSS came from 😆

Reply:
Whoops 😊 Maybe I should slim down the list of variables. I was hoping at some point Stephen or Silas could build us something nicer. 😄